c/src/nnstreamer-capi-pipeline.c

```diff
@@ -1076,6 +1076,41 @@ ml_pipeline_stop (ml_pipeline_h pipe)
   return ML_ERROR_NONE;
 }
 
+/**
+ * @brief Clears all data and resets the running-time of the pipeline (more info in nnstreamer.h)
+ */
+int
+ml_pipeline_flush (ml_pipeline_h pipe, bool start)
+{
+  ml_pipeline *p = pipe;
+  int status;
+
+  check_feature_state ();
+
+  if (p == NULL)
+    return ML_ERROR_INVALID_PARAMETER;
+
+  status = ml_pipeline_stop (pipe);
+  if (status != ML_ERROR_NONE)
+    return status;
+
+  ml_logi ("The pipeline is stopped, clear all data from the pipeline.");
+
+  /* send flush event to pipeline */
+  g_mutex_lock (&p->lock);
+  if (!gst_element_send_event (p->element, gst_event_new_flush_start ()) ||
```

Comment:
Error report:
There is a case when tensor_mux fails to `flush_start`
Might need to fix tensor_mux

example log:
W/GStreamer+collectpads: 0:00:06.914059007 0x79663112d0 ../libs/gst/base/gstcollectpads.c:1711:gst_collect_pads_event_default:<mux:sink_0> forwarding flush start failed
E/GStreamer+queue: 0:00:06.914213434 0x79663112d0 ../plugins/elements/gstqueue.c:1028:gst_queue_handle_sink_event:<queue0> Failed to push event
W/GStreamer+collectpads: 0:00:06.914267341 0x79663112d0 ../libs/gst/base/gstcollectpads.c:1711:gst_collect_pads_event_default:<mux:sink_1> forwarding flush start failed
E/GStreamer+queue: 0:00:06.914379216 0x79663112d0 ../plugins/elements/gstqueue.c:1028:gst_queue_handle_sink_event:<queue1> Failed to push event
W/GStreamer+collectpads: 0:00:06.914428226 0x79663112d0 ../libs/gst/base/gstcollectpads.c:1711:gst_collect_pads_event_default:<mux:sink_2> forwarding flush start failed
E/GStreamer+queue: 0:00:06.914507705 0x79663112d0

Reply:
We cannot assure that all elements in pipeline successfully handle flush event.
I will change this condition and print warning message.